pkg/controller/multi_unmapper_at.go

```diff
@@ -0,0 +1,78 @@
+package controller
+
+import (
+	"strings"
+	"sync"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/longhorn/longhorn-engine/pkg/types"
+)
+
+type MultiUnmapperAt struct {
+	lock      *sync.Mutex
+	unmappers []types.UnmapperAt
+}
+
+type MultiUnmapperError struct {
+	errors []error
+}
+
+func (m *MultiUnmapperError) Error() string {
+	errors := []string{}
+	for _, err := range m.errors {
+		if err != nil {
+			errors = append(errors, err.Error())
+		}
+	}
+
+	switch len(errors) {
+	case 0:
+		return "Unknown"
+	case 1:
+		return errors[0]
+	default:
+		return strings.Join(errors, "; ")
```

Comment:
You can just keep `default`.

Reply:
You can skip `case 1` because it will be just part of `default`.

Reply:
Got it. Updated.